src/Microsoft.SqlTools.ServiceLayer/Connection/Contracts/SecurityTokenRequest.cs

```diff
@@ -28,6 +28,12 @@ class RequestSecurityTokenParams
         /// Gets or sets the scope of the authentication request.
         /// </summary>
         public string Scope { get; set; }
+
+        /// <summary>
+        /// Gets or sets the account ID
+        /// </summary>
+        public string AccountId { get; set; }
```

Comment:
Why are you changing these notifications? You don't appear to be modifying the handlers for these

Reply:
I think I did this on accident, I will change it back